aspnetcore/fundamentals/openapi/samples/9.x/BuildTime/TestScripts/combined.ps1

```diff
@@ -0,0 +1,63 @@
+# Creates a Web API project, adds ApiDescription.Server
+# Use one arguement and it creates ../MyOpenApiTest.json
+cls
+## any arg sets the Inner_text to ".."
+if ($args.Count -gt 0) { 
+    $Inner_text = ".."
+} else {
+    $Inner_text = "./" 
+}
```

Comment:
@martincostello  using NuGet pkg 9.0.0-rc.1.24412.15
I'm still getting an error on `./`
`..` correctly moves it up to the parent directory.
Should I file a bug for this at https://github.com/dotnet/aspnetcore/issues ?

Reply:
I think maybe I got confused, and forgot that I was overriding the output location with [`OpenApiDocumentsDirectory`](https://github.com/martincostello/api/blob/78052239e21d9cf7dc896d7f61766ef7aa85292e/src/API/API.csproj#L11)